relay-dynamic-config/src/project.rs

```diff
@@ -62,6 +66,13 @@ impl Retentions {
                     None
                 }
             }
+            TraceItemType::Metric => {
+                if let Some(metric) = &self.metric {
+                    Some(metric)
+                } else {
+                    None
+                }
```

Comment:
Copy paste from above, but that's just:


```suggestion
                self.metric.as_ref()
```